data/campaigns/World_Conquest/lua/game_mechanics/effects.lua

```diff
@@ -66,12 +66,12 @@ end
 -- Like apply_to=resistance with replace=true, but never decreases resistances.
 function wesnoth.effects.wc2_min_resistance(u, cfg)
 	local resistance_new = {}
-	local resistance_old = wml.parsed(wml.get_child(cfg, "resistance"))
+	local resistance_old = wml.get_child(cfg, "resistance")
 	local unit_resistance_cfg = nil
 	for k,v in pairs(resistance_old) do
 		if type(k) == "string" and type(v) == "number" then
 			if not unit_resistance_cfg then
-				unit_resistance_cfg = wml.parsed(wml.get_child(u.__cfg, "resistance"))
+				unit_resistance_cfg = wml.get_child(u.__cfg, "resistance")
```

Comment:
Oh, this one too should be left as it is.

Reply:
Since this is the one that is causing an error, when `wml.parsed()` is restored, I did not re-add it. Now, WC map generation works fine without errors.

Reply:
Hmm, come to think of it, this is reading from `u.__cfg` so it's pointless to use `wml.parsed` anyway.